File: mobile/iOS/GuestJini/GuestJini/Global/Model/GroupConversation.swift
//
//  GroupConversation.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 30/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class GroupConversation: Codable, Identifiable, ObservableObject {
    public var id:String? = ""
    public var clientId:String?  = ""
    public var orgId:String? = ""
    public var groupId:String? = ""
    public var displayName:String? = ""
    public var message:String? = ""
    public var creationTime:String? = ""
    public var isItMe:Int? = 0
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/Group/GroupConversation.java
package com.carmel.guestjini.Networking.Group;

public class GroupConversation {
    private String id;
    private String clientId;
    private String orgId;
    private String groupId;
    private String displayName;
    private String message;
    private String creationTime;
    private int isItMe;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getClientId() {
        return clientId;
    }

    public void setClientId(String clientId) {
        this.clientId = clientId;
    }

    public String getOrgId() {
        return orgId;
    }

    public void setOrgId(String orgId) {
        this.orgId = orgId;
    }

    public String getGroupId() {
        return groupId;
    }

    public void setGroupId(String groupId) {
        this.groupId = groupId;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getCreationTime() {
        return creationTime;
    }

    public void setCreationTime(String creationTime) {
        this.creationTime = creationTime;
    }

    public int getIsItMe() {
        return isItMe;
    }

    public void setIsItMe(int isItMe) {
        this.isItMe = isItMe;
    }
}
